Composer/packages/types/src/shell.ts

```diff
@@ -67,12 +67,16 @@ export type DisabledMenuActions = {
 export type NotificationLink = { label: string; onClick: () => void };
 
 export type Notification = {
-  type: 'info' | 'warning' | 'error' | 'pending' | 'success' | 'question' | 'congratulation';
+  type: 'info' | 'warning' | 'error' | 'pending' | 'success' | 'question' | 'congratulation' | 'custom';
   title: string;
   description?: string;
   retentionTime?: number;
   link?: NotificationLink;
-  links?: NotificationLink[];
+  links?: (NotificationLink | null | undefined)[];
```

Comment:
Why are we adding null or undefined? Seems very unnecessary.

Reply:
It is, now that I've split them into left and right groups.